Testing/VelaptorTests/Fakes/GPUBufferFake.cs

```diff
@@ -58,6 +59,16 @@ internal sealed class GPUBufferFake : GPUBufferBase<TextureBatchItem>
     /// <remarks>Used for unit testing.</remarks>
     public bool UpdateVertexDataInvoked { get; private set; }
 
+    /// <summary>
+    /// Gets the view port size for the purpose of testing.
```

Comment:
Refactor viewport to one word
```suggestion
    /// Gets the viewport size for the purpose of testing.
```